### ISSUE TITLE
Phantom Steal Has Phantom'd Itself Out Of Existence!

### ISSUE THREAD

**Issue by Astralika** [plasmaster09/Better-BR-Jester#14]
Or... More accurately, the keymaster's lock has locked it away.

When I used Phantom Steal on the keymaster and subsequently used the Lock 4+, it replaced Phantom Steal with Lock 4+ in my deck for the rest of the run. I assume this has something to do with the transformation effect of Lock 4+.

**Comment by plasmaster09** [comment]
Yep. I believe that'd happen in vanilla with CFB Thief, and is a bug pertaining to recursive transformation in general, so it's not exactly a thing I can fix- it's just that having a copying card from the start makes the bug more common to encounter.
